vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -5,24 +5,30 @@ sidebarDepth: 2
 # Entando Hub
 ## Overview
 
-The Entando Hub enables a team to share components across Entando Applications. It can be installed in Entando 6.3.2 or higher and includes API-level integration with the Entando 7.0 App Builder.
+The enterprise Entando Hub enables teams to share components across the organization and between Entando Applications. It can be installed in Entando 6.3.2 or higher and includes API-level integration with the Entando 7.0 App Builder.
 
 The Hub allows users to:
 
 - Centralize components and business capabilities for use across teams, groups, or clients.
 - Publish, manage and communicate component features, versions and metadata.
 - Perform business-level assessment of component readiness. 
 
-This tutorial covers:
+An Entando Application can make use of an Entando Hub in several ways:
+* The Local Hub, included in the Entando App Builder, displays a collection of components ready for use. These can be used to compose an application or as a starting point to create new components.	
+* Entando Cloud Hub is the public catalog containing packaged business capabilities and components provided by Entando and its partners throughout the world.
+* An enterprise Entando Hub, developed and curated by Entando clients and partners, can be used to share components within their organization or made available  for public use.
+
+This tutorial covers the steps for an enterprise Entando Hub:
```

Comment:
I have mixed feelings on "enterprise" here. I feel like the instructions are general purpose and since we don't have instructions for #1 (since it's baked in) or #2 (since it's ours), calling out #3 adds a little confusion.